DigitalLearningSolutions.Data/Services/TrackerActionService.cs

```diff
@@ -74,22 +74,41 @@ public TrackerEndpointResponse StoreDiagnosticJson(int? progressId, string? diag
                 return TrackerEndpointResponse.StoreDiagnosticScoreException;
             }
 
-            var diagnosticOutcomes = JsonConvert.DeserializeObject<IEnumerable<DiagnosticOutcome>>(diagnosticOutcome);
             try
             {
-                foreach (var diagOutcome in diagnosticOutcomes)
+                var diagnosticOutcomes = JsonConvert.DeserializeObject<IEnumerable<DiagnosticOutcome>>(
+                    diagnosticOutcome,
+                    new JsonSerializerSettings
+                    {
+                        MissingMemberHandling = MissingMemberHandling.Error,
+                        NullValueHandling = NullValueHandling.Include,

```

Comment:
`NullValueHandling.Include` and `DefaultValueHandling.Include` are already the default values for these settings so these can be dropped from here.

Reply:
oops, was playing around with some other settings and forgot to remove them